packages/hub-score/README.md

```diff
@@ -0,0 +1,70 @@
+
+
+## Description
+
+This project aims to extract the credit score of each wallet address entered,and ensure that it qualifies for a loan.
```

Comment:
This is not a project - rewrite this. 
It is not extracting credit score neither - we are talking about stats for borrowers here based on their loans. Their is not credit score.

Reply:
this is okey "This project aims to creates stadistic of each wallet address entered,for loans."